rohmu/notifier/null.py

```diff
@@ -0,0 +1,19 @@
+"""Copyright (c) 2022 Aiven, Helsinki, Finland. https://aiven.io/"""
+from .interface import Notifier
+from typing import Optional
+
+
+class NullNotifier(Notifier):
+    """Empty implementation.
+
+    Used by default if configuration is missing to avoid None checks
+    """
+
+    def object_created(self, key: str, size: Optional[int]) -> None:
+        pass
+
+    def object_deleted(self, key: str) -> None:
+        pass
+
+    def tree_deleted(self, key: str) -> None:
+        pass
```

Comment:
Should we create a (singleton) instance of this one that could be used a default value for notifier params (avoiding `Optional`s)

Reply:
There is no need for a singleton, I initialized the value on the `BaseTransfer` class:

https://github.com/aiven/rohmu/pull/67/files#diff-19b8216f21f883fff1a37d854110c70b40006e6be762e8251178b8f654247285R30